tests/network/localnet/migration_stuntime/test_migration_stuntime.py

```
@@ -19,35 +19,36 @@
-            pytest.param(6, id="ipv6", marks=[pytest.mark.polarion("CNV-15272"), pytest.mark.ipv6]),
-        ],
-        indirect=True,
-    )
-    def test_client_migrates_off_server_node(
+class MigrationStuntimeBase:
+    """Base class with migration stuntime test implementations.
+
+    Subclasses must set ``IP_FAMILY`` (4 or 6) and apply per-test Polarion markers.
+    """
+
+    def test_client_migrates_off_server_node(  # noqa: PID001 - Polarion IDs are on the subclass overrides
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: PID001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)